languagetool-core/src/test/java/org/languagetool/rules/CommaWhitespaceRuleTest.java

```diff
@@ -95,7 +95,17 @@ public void testRule() throws IOException {
     matches = rule.match(lt.getAnalyzedSentence("This , is a test sentence."));
     assertEquals(1, matches.length);
     assertEquals(",", matches[0].getSuggestedReplacements().get(0));
-    
+
+    matches = rule.match(lt.getAnalyzedSentence("You \" fixed\" it."));
+    assertEquals(1, matches.length);
+    assertEquals("\"", matches[0].getSuggestedReplacements().get(0));
+    assertEquals(4, matches[0].getFromPos());
+    assertEquals(6, matches[0].getToPos());
+    matches = rule.match(lt.getAnalyzedSentence("You \"fixed \" it."));
+    assertEquals(1, matches.length);
+    assertEquals("\"", matches[0].getSuggestedReplacements().get(0));
+    assertEquals(11, matches[0].getFromPos());
+    assertEquals(13, matches[0].getToPos());
```

Comment:
This should be 10–13 if we match both spaces. The current value is not good, as it attaches the quote to the following word instead of the one before.